corehq/util/datadog/utils.py

```diff
@@ -82,3 +82,20 @@ def get_url_group(url):
 def update_datadog_metrics(metrics):
     for metric, value in metrics.items():
         statsd.gauge(metric, value)
+
+
+def bucket_value(value, buckets, unit=None):
+    """Get value bucket for the given value
+
+    Bucket values because datadog's histogram is too limited
+
+    Basically frequency is not high enough to have a meaningful
+    distribution with datadog's 10s aggregation window, especially
+    with tags. More details:
+    https://help.datadoghq.com/hc/en-us/articles/211545826
+    """
+    unit = unit or ''
```

Comment:
strings are immutable. can just set `''` as the default value rather than `None`.